src/Sulu/Bundle/MediaBundle/Resources/public/js/services/media-manager.js

```diff
@@ -109,6 +111,11 @@ define([
                 promise.resolve(media.toJSON());
             } else {
                 media = Media.findOrCreate({id: mediaId});
+                if (clear) {
```

Comment:
Do we really need a flag here? Is there any case where this behavior is what we want?